powermon/commands/command.py

```diff
@@ -113,7 +113,6 @@ def to_dto(self):
         return CommandDTO(
             command_code = self.code,
             device_id=self.device_id,
-            result_topic = self.outputs[0].get_topic(),
```

Comment:
The result topic is needed by the API to subscribe to command results. The abstract_output has a get_topic defined so other outputs shouldn't be impacted. What error caused you to remove this?

Reply:
The error was due to missed method in MQQT class